pkgs/tools/text/jumanpp/default.nix

```diff
@@ -9,7 +9,22 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ASdr6qbkSe71M7QmuuwidCa4xQhDVoXBJ2XqvSY53pQ=";
   };
 
-  patches = [ ./0001-Exclude-all-tests-from-the-build.patch ];
+  patches = [
+    ./0001-Exclude-all-tests-from-the-build.patch
+    # https://github.com/ku-nlp/jumanpp/pull/132
+    (fetchpatch {
+      url = "https://github.com/ku-nlp/jumanpp/commit/cc0d555287c8b214e9d6f0279c449a4e035deee4.patch";
```

Comment:
This patch does not belong to the mentioned PR?
Please add descriptive names to the patches, so that their purpose is identifiable.

Reply:
Without this patch, the third fetchpatch would be rejected.